pages/vi/profiles/evoong.md

```diff
@@ -0,0 +1,16 @@
+## About Me
+
+- **Name:** Eric Voong
+- **Location:** Vaughan Ontario
+- **Time Zone:** EST
+- **OS:** Windows 10 Home Version 1909
+
+## Description:
+⋅⋅* I am a second year Nanotechnology Student from the Unviersity of Waterloo.
+⋅⋅* It is a pleasure meet you! I like solving, designing, and creating with a passion. 
+⋅⋅* I am someone who gets absorbed into my projects as time passes by and loves exceeding expectations. 
+⋅⋅* My goal is to work on a service or product that can make a change in a persons everyday life and have an impact on the world. 
+
+![My Linkedin](https://www.linkedin.com/in/eric-voong/)
```

Comment:
You should change this markdown element to a link, not an image.

Reply:
done